package.json

```diff
@@ -55,8 +55,8 @@
     "cross-env": "^6.0.0",
     "kcd-scripts": "^1.7.0",
     "npm-run-all": "^4.1.5",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
+    "react": "^0.0.0-experimental-f6b8d31a7",
+    "react-dom": "^0.0.0-experimental-f6b8d31a7",
```

Comment:
I think that this resolves to the latest stable version. If we want to support the experimental release we have to do an exact version rather than `^`. I believe that's why tests are failing.